.github/workflows/pr-validation.yml

```diff
@@ -20,6 +20,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2.3.4
+    - name: Setup PHP and Xdebug for Code Coverage report
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: '7.x'
```

Comment:
```suggestion
        php-version: '7.3'
```

Can we possibly put 7.3 since that's the minimum PHP version we use.

Reply:
Updated.